tests/Feature/Data/Reader/EntityReaderTest.php

```diff
@@ -111,6 +114,15 @@ public function testLimit(): void
         );
     }
 
+    /**
+     * Test {@see EntityReader::withLimit()}
```

Comment:
Might be replaces with `@covers \Yiisoft\Yii\Cycle\Data\Reader\EntityReader::withLimit`
or
```php
/**
 * @coversDefaultClass \Yiisoft\Yii\Cycle\Data\Reader\EntityReader
 */
class ClassTest {
    /**
     * @covers ::withLimit
     */
    public function testMethod() {}
}
```